metal/label_model/class_balance.py

```diff
@@ -0,0 +1,136 @@
+from itertools import product
+
+import numpy as np
+import torch
+from torch import nn, optim
+
+
+class ClassBalanceModel(nn.Module):
```

Comment:
Especially since this is under the label_model/ directory, I think it's probably worth an extra disclaimer in the docstring that this is _not_ a LabelModel. You do very clearly define what it's actually computing, but I'm sure someone will try to use it the wrong way otherwise.

Reply:
It actually is one, as it implicitly estimates the accuracies of the LFs, *for conditionally independent LFs.*  How exactly it fits into the overall repo, therefore, may be a little bit fluid... but I want to get it in at least in this separate form now, so can at least be used for class balance estimation